package.json

```diff
@@ -4,27 +4,32 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
-    "date-fns": "latest",
```

Comment:
Pretty sure we need this for the date picker

Reply:
It is still there check the package.json